framework/src/main/java/org/checkerframework/common/value/ValueVisitor.java

```diff
@@ -29,7 +29,7 @@
 import org.checkerframework.framework.type.AnnotatedTypeMirror;
 import org.checkerframework.framework.type.AnnotatedTypeMirror.AnnotatedDeclaredType;
 import org.checkerframework.framework.type.visitor.AnnotatedTypeScanner;
-import org.checkerframework.javacutil.AnnotationUtils;
+import static org.checkerframework.javacutil.AnnotationUtils.*;
```

Comment:
In general we avoid wildcard imports and there are very few static imports.
Please only import the methods you actually use.

Reply:
Can do. Next commit will have it split.